### PR TITLE
HLM-5195

### PR DESCRIPTION
**Pull request by siddhant-nawale-egov**
Adding Pagination icons and Fixing Population and Facility icons
Adding close button

## Summary by CodeRabbit

- **New Features**
    - Added a Close button and Loader to the InboxSearchComposer component.
    - Introduced Pagination icons to the SVG components library.

- **Enhancements**
    - Expanded export list to include CloseButton component.
    - Improved visual representation of the Population SVG component.

- **Documentation**
    - Updated SVG components library documentation to reflect new version and changes.

### CODE REVIEW

**Review by nipunarora-eGov — COMMENTED** [review]
@siddhant-nawale-egov @jagankumar-egov Do we need to update react-comp package version or are we running it locally?

**Comment by siddhant-nawale-egov** [comment]
> @siddhant-nawale-egov @jagankumar-egov Do we need to update react-comp package version or are we running it locally?

Yes we need to update the react-comp package version. Updating it.

File: micro-ui/web/micro-ui-internals/packages/svg-components/src/index.js [hunk 1]
```diff
@@ -1577,5 +1580,9 @@ export {
   TickMarkBackgroundFilled,
   Excel,
   SpatialDocument,
-  Trash
+  Trash,
+  PaginationPrevious,
+  PaginationNext,
+  PaginationLast,
+  PaginationFirst,
```
[the remark this answers is not in the document]
Comment (siddhant-nawale-egov) [reply micro-ui/web/micro-ui-internals/packages/svg-components/src/index.js]: I checked the code but i dint find any duplications.

File: micro-ui/web/micro-ui-internals/packages/react-components/README.md [hunk 2]
```diff
@@ -137,6 +137,7 @@ To use the InboxSearchComposer component for managing multiple tabs, follow thes
 ### Changelog
 
 ```bash
+1.8.1-beta.8 Added Close button, Custom Modal and Loader
```
[the remark this answers is not in the document]
Comment (siddhant-nawale-egov) [reply micro-ui/web/micro-ui-internals/packages/react-components/README.md:140]: I dont think that is applicable here

**Pull request merged**